src/EnergyPlus/PlantLoadProfile.cc

```diff
@@ -353,24 +414,55 @@ void GetPlantProfileInput(EnergyPlusData &state)
 
-            state.dataPlantLoadProfile->PlantProfile(ProfileNum).InletNode = GetOnlySingleNode(state,
-                                                                                               state.dataIPShortCut->cAlphaArgs(2),
-                                                                                               ErrorsFound,
-                                                                                               DataLoopNode::ConnectionObjectType::LoadProfilePlant,
-                                                                                               state.dataIPShortCut->cAlphaArgs(1),
-                                                                                               DataLoopNode::NodeFluidType::Water,
-                                                                                               DataLoopNode::ConnectionType::Inlet,
-                                                                                               NodeInputManager::CompFluidStream::Primary,
-                                                                                               ObjectIsNotParent);
-            state.dataPlantLoadProfile->PlantProfile(ProfileNum).OutletNode = GetOnlySingleNode(state,
-                                                                                                state.dataIPShortCut->cAlphaArgs(3),
-                                                                                                ErrorsFound,
-                                                                                                DataLoopNode::ConnectionObjectType::LoadProfilePlant,
-                                                                                                state.dataIPShortCut->cAlphaArgs(1),
-                                                                                                DataLoopNode::NodeFluidType::Water,
-                                                                                                DataLoopNode::ConnectionType::Outlet,
-                                                                                                NodeInputManager::CompFluidStream::Primary,
-                                                                                                ObjectIsNotParent);
+            if (UtilityRoutines::SameString(state.dataIPShortCut->cAlphaArgs(6), "Steam")) {
+                state.dataPlantLoadProfile->PlantProfile(ProfileNum).FluidType = PlantLoopFluidType::Steam;
+            } else {
+                state.dataPlantLoadProfile->PlantProfile(ProfileNum).FluidType = PlantLoopFluidType::Water;
+            }
+
+            if (state.dataPlantLoadProfile->PlantProfile(ProfileNum).FluidType == PlantLoopFluidType::Water) {
+                state.dataPlantLoadProfile->PlantProfile(ProfileNum).InletNode =
+                    GetOnlySingleNode(state,
```

Comment:
Why do we not get both `Inlet` and `Outlet` nodes in a single function call? This is a serious question. Maybe not for you, @dareumnam, but in general.